Some module has a round error similar to #975.

<!--
If you have a question rather than reporting a bug please go to http://answers.opencv.org where you get much faster responses.
If you need further assistance please read [How To Contribute](https://github.com/opencv/opencv/wiki/How_to_contribute).

This is a template helping you to create an issue which can be processed as quickly as possible. This is the bug reporting section for the OpenCV library.
-->

##### System information (version)
<!-- Example
- OpenCV => 3.1
- Operating System / Platform => Windows 64 Bit
- Compiler => Visual Studio 2015
-->

- OpenCV => 3.2(master)
- Operating System / Platform => Windows 7 64 Bit
- Compiler => Visual Studio 2012

##### Detailed description

<!-- your description -->

#975
This is an error list.
So can I fix it?

`Error	318	error C3861: 'round': identifier not found	C:\opencv_contrib\modules\ximgproc\src\fast_line_detector.cpp	185	1	opencv_ximgproc
Error	319	error C3861: 'round': identifier not found	C:\opencv_contrib\modules\ximgproc\src\fast_line_detector.cpp	186	1	opencv_ximgproc
Error	320	error C3861: 'round': identifier not found	C:\opencv_contrib\modules\ximgproc\src\fast_line_detector.cpp	188	1	opencv_ximgproc
Error	321	error C3861: 'round': identifier not found	C:\opencv_contrib\modules\ximgproc\src\fast_line_detector.cpp	720	1	opencv_ximgproc
Error	322	error C3861: 'round': identifier not found	C:\opencv_contrib\modules\ximgproc\src\fast_line_detector.cpp	721	1	opencv_ximgproc
Error	323	error C3861: 'round': identifier not found	C:\opencv_contrib\modules\ximgproc\src\fast_line_detector.cpp	724	1	opencv_ximgproc
Error	324	error C3861: 'round': identifier not found	C:\opencv_contrib\modules\ximgproc\src\fast_line_detector.cpp	725	1	opencv_ximgproc
Error	325	error C3861: 'round': identifier not found	C:\opencv_contrib\modules\ximgproc\src\fast_line_detector.cpp	727	1	opencv_ximgproc
Error	340	error C3861: 'round': identifier not found	C:\opencv_contrib\modules\text\src\ocr_hmm_decoder.cpp	985	1	opencv_text
Error	341	error C3861: 'round': identifier not found	C:\opencv_contrib\modules\text\src\ocr_beamsearch_decoder.cpp	569	1	opencv_text
`

--- a/modules/text/src/ocr_beamsearch_decoder.cpp
+++ b/modules/text/src/ocr_beamsearch_decoder.cpp
@@ -566,7 +566,7 @@ OCRBeamSearchClassifierCNN::OCRBeamSearchClassifierCNN (const string& filename)
 
     nr_feature = weights.rows;
     nr_class   = weights.cols;
-    patch_size  = (int)round(sqrt((float)kernels.cols));
+    patch_size  = cvRound(sqrt((float)kernels.cols));
     window_size = 4*patch_size;
     step_size   = 4;
     quad_size   = 12;

--- a/modules/text/src/ocr_hmm_decoder.cpp
+++ b/modules/text/src/ocr_hmm_decoder.cpp
@@ -982,7 +982,7 @@ OCRHMMClassifierCNN::OCRHMMClassifierCNN (const string& filename)
 
     nr_feature  = weights.rows;
     nr_class    = weights.cols;
-    patch_size  = (int)round(sqrt((float)kernels.cols));
+    patch_size  = cvRound(sqrt((float)kernels.cols));
     // algorithm internal parameters
     window_size = 32;
     num_quads   = 25;

--- a/modules/ximgproc/src/fast_line_detector.cpp
+++ b/modules/ximgproc/src/fast_line_detector.cpp
@@ -182,10 +182,10 @@ void FastLineDetectorImpl::drawSegments(InputOutputArray _image, InputArray line
             getAngle(seg);
             double ang = (double)seg.angle;
             Point2i p1;
-            p1.x = (int)round(seg.x2 - gap*cos(arrow_angle * CV_PI / 180.0 + ang));
-            p1.y = (int)round(seg.y2 - gap*sin(arrow_angle * CV_PI / 180.0 + ang));
+            p1.x = cvRound(seg.x2 - gap*cos(arrow_angle * CV_PI / 180.0 + ang));
+            p1.y = cvRound(seg.y2 - gap*sin(arrow_angle * CV_PI / 180.0 + ang));
             pointInboardTest(_image.getMatRef(), p1);
-            line(_image.getMatRef(), Point((int)round(seg.x2), (int)round(seg.y2)), p1, Scalar(0,0,255), 1);
+            line(_image.getMatRef(), Point(cvRound(seg.x2), cvRound(seg.y2)), p1, Scalar(0,0,255), 1);
         }
     }
 }
@@ -717,14 +717,14 @@ void FastLineDetectorImpl::drawSegment(Mat& mat, const SEGMENT& seg, Scalar bgr,
     double arrow_angle = 30.0;
 
     Point2i p1;
-    p1.x = (int)round(seg.x2 - gap*cos(arrow_angle * CV_PI / 180.0 + ang));
-    p1.y = (int)round(seg.y2 - gap*sin(arrow_angle * CV_PI / 180.0 + ang));
+    p1.x = cvRound(seg.x2 - gap*cos(arrow_angle * CV_PI / 180.0 + ang));
+    p1.y = cvRound(seg.y2 - gap*sin(arrow_angle * CV_PI / 180.0 + ang));
     pointInboardTest(mat, p1);
 
-    line(mat, Point((int)round(seg.x1), (int)round(seg.y1)),
-            Point((int)round(seg.x2), (int)round(seg.y2)), bgr, thickness, 1);
+    line(mat, Point(cvRound(seg.x1), cvRound(seg.y1)),
+            Point(cvRound(seg.x2), cvRound(seg.y2)), bgr, thickness, 1);
     if(directed)
-        line(mat, Point((int)round(seg.x2), (int)round(seg.y2)), p1, bgr, thickness, 1);
+        line(mat, Point(cvRound(seg.x2), cvRound(seg.y2)), p1, bgr, thickness, 1);
 }
 } // namespace cv
 } // namespace ximgproc

